lib/govuk_publishing_components/presenters/component_wrapper_helper.rb

```diff
@@ -76,6 +78,11 @@ def set_tabindex(tabindex_attribute)
         @options[:tabindex] = tabindex_attribute
       end
 
+      def set_dir(dir_attribute)
```

Comment:
I'm starting to wonder if there's a Ruby way of writing a generic method here to save repeating essentially the same method. Not for this PR, but something to ponder, maybe something like:

```Ruby
def set_thing(method_name, attribute)
  # call function "check_#{method_name}"_is_valid(attribute)
  @options[method_name.to_key] = attribute
end
```

Reply:
Good point, I noticed this as well so will have a think 👍